locales/en/discord.js

```diff
@@ -0,0 +1,17 @@
+export default {
+	INSUFFICIENT_PERMISSIONS: {
+		USER: 'You are missing permission(s): %1',
+		BOT: {
+			DEFAULT: 'I am missing permission(s): %1',
+			BASIC: 'I need to be able to connect and speak in the voice channel.',
+			STAGE: 'I need to be a stage moderator in the stage channel.',
+		},
+	},
+	TIMED_OUT: 'I am currently timed out.',
```

Comment:
Should `TIMED_OUT` be under `INSUFFICIENT_PERMISSIONS.BOT`?

Reply:
could you help me check if all the usages of this is always to do with insufficient permissions?

Reply:
Usages of `DISCORD.TIMED_OUT`:
commands/play.js
components/selectmenus/play.js

Reply:
yea, I don't mind shifting it over